changelogs/fragments/831-proxmox-kvm-wait.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - proxmox_kvm - improve handling of long-running tasks by creating a dedidacted function (https://github.com/ansible-collections/community.general/pull/831).
```

```suggestion
  - proxmox_kvm - improve handling of long-running tasks by creating a dedicated function (https://github.com/ansible-collections/community.general/pull/831).
```
Sorry, didn't spot that earlier...